src/shared/lib/comparison/ComparisonStoreUtils.ts

```diff
@@ -189,7 +189,8 @@ export const cnaEventTypeSelectInit: {
 export function mutationEventTypeSelectInit() {
     const object = {} as { [key in MutationEnrichmentEventType]?: boolean };
     Object.keys(MutationEnrichmentEventType).forEach(type => {
-        object[type as MutationEnrichmentEventType] = true;
+        object[type as MutationEnrichmentEventType] =
```

Comment:
not sure i understand this logic.  it's unselected if it's of type any?    comment would be good to explain why.  but i can merge.

Reply:
Not exactly. `MutationEnrichmentEventType.any` is never used/updated in frontend, but its default value is set to `true`. With my recent backend api changes I am trying to leverage this option to filter data.

If `any` is set to `true` then no mutation type filtering is done. If it is `false` filtering is done based on the other mutation type selection

Similarly for `other` option, which is used in the frontend has a default value `true` - meaning to include other mutation types.